### PR TITLE
Improve PropertyHelper documentation

### PR DESCRIPTION
**Pull request by dchambers**

### CODE REVIEW

File: brjs-sdk/sdk/libs/javascript/br-presenter/src/br/presenter/property/PropertyHelper.js [hunk 1]
```diff
@@ -7,11 +7,18 @@
 /**
  * @class
  * @alias module:br/presenter/property/PropertyHelper
- * 
+ *
  * @classdesc
- * The PropertyHelper is a utility to simplify the attaching and monitoring of property listeners. 
- * <p>This is a useful class if you want to keep track of all the properties of certain node in a single place. It
- * will also provide some methods that give you extra functionality for handling call-backs from your properties.</p>
+ * <code>PropertyHelper</code> is a utility that simplifies the attaching and removing of property listeners. Compared
```
Comment (thecapdan) [inline brjs-sdk/sdk/libs/javascript/br-presenter/src/br/presenter/property/PropertyHelper.js]: "compared with"? how about "In addition to" or "as well as"?

Comment (dchambers) [inline brjs-sdk/sdk/libs/javascript/br-presenter/src/br/presenter/property/PropertyHelper.js]: Is this clearer:

> Compared with adding and removing listeners on `Property` directly, using the equivalent methods on the `PropertyHelper` class offers a number of benefits:

Comment (thecapdan) [inline brjs-sdk/sdk/libs/javascript/br-presenter/src/br/presenter/property/PropertyHelper.js]: much clearer :+1:

Comment (dchambers) [inline brjs-sdk/sdk/libs/javascript/br-presenter/src/br/presenter/property/PropertyHelper.js]: Okay @thecapdan, I've now made that change.

**Pull request merged**